Documentation/concepts/networking/routing.rst

```diff
@@ -61,7 +61,8 @@ Identity context
   Encapsulation protocols allow for the carrying of metadata along with the
   network packet. Cilium makes use of this ability to transfer metadata such as
   the source security identity and load balancing state to perform
-  direct-server-return (DSR).
+  direct-server-return (DSR). The security identity transfer is an optimization
+  designed to avoid one identity lookup on the remote node.
```

Comment:
@brb Is the same true for `load balancing state`?

Reply:
Nice catch! After we had removed the `bpf/bpf_lb.c`, this became no longer true. Currently, we don't transfer any DSR-related state in an encapsulation metadata. However, this might change if we switch from custom IPv{4,6} options to IP-IP encapsulation to implement DSR. For now, we can remove the mentioning of LB+DSR.